### ISSUE TITLE
add a simple / dummy CI pipeline

### ISSUE THREAD

**Issue by rawlingsj** [openshiftio/saas-openshiftio#230]
fabric8 pipelines will now only auto merge PRs providing CI tests have passed which is working great.  The issue here is this repo has no CI on it so the commit status remains pending and so tenant + forge pipelines are blocked.  

Are there any objections from anyone if I add a dummy Jenkinsfile to this repo so commits in PRs will get a success status?  It's a quick win else we'll have to add logic to the pipeline library to work out if a repo has CI enabled on it.  RIght now I don't know of a good way to do that so I'd rather not block the release pipelines for now.

@vpavlin @kbsingh any objections?

**Comment by rawlingsj** [comment]
@vpavlin @kbsingh if you're happy just merge this PR and also could one of you add a webhook on this repo..

Payload URL:  https://jenkins.cd.test.fabric8.io/github-webhook
Content type: application/json
Which events would you like to trigger this webhook?: Send Everything
and select

**Comment by vpavlin** [comment]
I'd rather have the PR built in CentOS CI - just as a matter of consistency - as the push to master runs there. 

I can add a job for that. I think I can even come up with a reasonable "test" for the PR - we can run a dry-run for what happens in master job.

**Comment by rawlingsj** [comment]
All Jobs are moving to these pipelines so I don't really see the need to do this in ci centos especially as this Job is ready to go.

**Comment by rawlingsj** [comment]
i.e. I've already done the work, why repeat it in a CI system that we're moving away from?

**Comment by vpavlin** [comment]
My understanding is that we are not moving away for next couple months at least. As I mentioned, I'd like to have my builds consistent which means one CI system per repository:) Also my solution to this problem actually runs a test which simulates prod deployment - thanks for pushing me to finally do it:) 

It's done - the only thing missing is centos-ci bot accepting invitation and then it's gonna start working.

**Comment by kbsingh** [comment]
couple of things to think through : 
* the actual deploy to prod runs on cico - we are not moving away from that till the alternative is stable and validated, we are quite a way from that. 

* since this repo is the approval step, we need a good way to pause and queue jobs etc - how would that work ? ideally this would mean the PR queue gets backlog'ed so the process can be run through when the cause-for-pause clears through.

**Comment by rawlingsj** [comment]
This issue is related to unblocking the the tenant and forge release pipelines.  Happy to discuss the approval steps and approach on another issue but right now I just need a CI job to update the PR status.

This PR + the webhooks described above will fix the issue and I really fail to understand the resistance seeing that we now use Jenkinsfiles on all UI, forge, tenant repos.  We are working towards building OSIO on OSIO but that is going to be harder of we keep adding jobs to ci centos instead of dogfooding our own tech stack.

If you still really feel strong about this then please do as you see fit, duplicate the work and create the job in ci centos. My main concern is unblocking the release pipelines so please either merge + do the webhooks (or give me karma to this repo) or close this and set up it up on ci centos, either way can we do this sooner rather than later please :)

**Comment by vpavlin** [comment]
As I mentioned - all is done, the only thing missing is centos-ci bot being able to submit approval, that's going to be fixed as soon as someone from Centos CI accepts the invitation, then things will start working:)

**Comment by vpavlin** [comment]
@rawlingsj Test's are passing after oc update on CI slave. I'd say the issue is solved.
